src/App.tsx

```diff
@@ -1,61 +1,196 @@
 import './App.scss';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import Button from '@mui/material/Button';
+import SendIcon from '@mui/icons-material/Send';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import FormHelperText from '@mui/material/FormHelperText';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import Select from '@mui/material/Select';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, User } from './types/types';
+import { TodoList } from './components/TodoList';
+
+const findUserById = (id: number, users: User[]) => {
+  return users.find(user => user.id === id) || null;
+};
+
+const requiredTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUserById(todo.userId, usersFromServer),
```

Comment:
maybe we can move usersFromServer upper (to our function findUserById) but i'm not sure.